appveyor.yml

```diff
@@ -4,7 +4,7 @@ branches:
     - master
     - RC_1_2
     - RC_1_1
-os: Visual Studio 2015
+image: Visual Studio 2015
```

Comment:
a few days back I found that `os` is not documented here: https://www.appveyor.com/docs/appveyor-yml